### PR TITLE
feat: bump express to ^4.19.2

### PR DESCRIPTION
**Pull request by alcpereira**
Due to [CVE-2024-29041](https://nvd.nist.gov/vuln/detail/CVE-2024-29041), we should bump express to version 4.19.2 or higher.

FYI I haven't tested yet the changes as I'm unsure how to properly do it.

### CODE REVIEW

**Comment by alcpereira** [comment]
> Thank you for the PR! Since this changes dependencies in source code would you mind branching from and repointing this to the `dev` branch?

Done 🙏

**Comment by brophdawg11** [comment]
Thank you - I forgot we have some flux in our templates between `main` and `dev` right now (`main` templates have been converted to be Vite, but `dev` is still `esbuild`).  I'm going to wait for `2.9.0` stable to be released which will sync those up and then we'll get this merged and avoid any weird merge conflicts.

File: .changeset/afraid-bears-fry.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+"integration-tests": patch
+"remix": patch
+"@remix-run/dev": patch
+"@remix-run/express": patch
+"@remix-run/serve": patch
+---
+
+feat: bump express to ^4.19.2
```
Comment (brophdawg11) [inline .changeset/afraid-bears-fry.md:9]:
```suggestion
Upgrade `express` dependency to `^4.19.2`
```

File: .changeset/afraid-bears-fry.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+"integration-tests": patch
+"remix": patch
+"@remix-run/dev": patch
+"@remix-run/express": patch
+"@remix-run/serve": patch
```
Comment (brophdawg11) [inline .changeset/afraid-bears-fry.md:6]:
```suggestion
"@remix-run/dev": patch
"@remix-run/express": patch
"@remix-run/serve": patch
```

**Comment by brophdawg11** [comment]
Thanks!

**Pull request merged**